examples/example_brownian.py

```diff
@@ -0,0 +1,62 @@
+from parcels import Grid, ScipyParticle
+import numpy as np
+from datetime import timedelta as delta
+import math
+import random
+
+
+def two_dim_brownian_flat(particle, grid, time, dt):
+
+    # random.seed() - should a seed be included for reproducibility/testing purposes?
+    # Use equation for particle diffusion.
+    particle.lat += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lat)
+    particle.lon += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lon)
+
+
+def brownian_grid(xdim=200, ydim=200):     # Define a flat grid of zeros, for simplicity.
+    depth = np.zeros(1, dtype=np.float32)
+    time = np.linspace(0., 100000. * 86400., 2, dtype=np.float64)
+
+    lon = np.linspace(0, 600000, xdim, dtype=np.float32)
+    lat = np.linspace(0, 600000, ydim, dtype=np.float32)
+
+    U = np.zeros((lon.size, lat.size, time.size), dtype=np.float32)
+    V = np.zeros((lon.size, lat.size, time.size), dtype=np.float32)
+
+    return Grid.from_data(U, lon, lat, V, lon, lat, depth, time, mesh='flat')
+
+
+def test_brownian_example(npart=3000):
+
+    grid = brownian_grid()
+    grid.K_lat = 100                  # Set diffusion constants.
+    grid.K_lon = 100
```

Comment:
@mlange05 what do you think of this idea to add extra variables to the `grid` via `grid.K = 100`? In this case, we need to pass a diffusion coefficient for use in the `two_dim_brownian_flat` kernel, and since arguments for kernels are fixed, this is the way around that @Thomas-95 and I came up with. Good? Or do you have a better idea?